extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -502,10 +515,34 @@ export class AmpStoryEmbeddedComponent {
     this.focusedStateOverlay_.addEventListener('click', event =>
       this.onOutsideTooltipClick_(event)
     );
+    this.tooltip_.addEventListener(
+      'click',
+      () => this.tooltipAnalytics_(),
+      true /** capture */
+    );
 
     return this.shadowRoot_;
   }
 
+  /**
+   * Configures analytics variables and fires analytic event.
+   * @private
+   */
+  tooltipAnalytics_() {
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.TOOLTIP_TARGET,
+      this.triggeringTarget_.tagName.toLowerCase()
+    );
+
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.TOOLTIP_TARGET_ATTRIBUTE,
+      this.triggeringTarget_.getAttribute(
+        this.getEmbedConfigFor_(this.triggeringTarget_).trackingAttribute
```

Comment:
Is this a standard way of doing the analytics tracking?  This seems kind of odd to me to have the component choose its attribute

Reply:
Would you prefer if we dynamically added an id of some sort to all components and used that instead?

At the bottom line we would have to do the same thing basically since we would have to check what that "id" for each component is. e.g. `data-tweetid` for twitter, `href` for a link, etc...